src/components/Dmitry/Dmitry.tsx

```diff
@@ -156,97 +226,110 @@ const dmitry = () => {
+                                checked={checkedDomainname}
+                                onChange={(e) => setCheckedDomainname(e.currentTarget.checked)}
+                            />
+                            <Checkbox
+                                label={"Retrieve Netcraft.com information on a host."}
+                                checked={checkedNetcraft}
+                                onChange={(e) => setCheckedNetcraft(e.currentTarget.checked)}
+                            />
+                            <Checkbox
+                                label={"Search for a possible subdomains."}
+                                checked={checkedSubdomains}
+                                onChange={(e) => setCheckedSubdomains(e.currentTarget.checked)}
+                            />
+                            <Checkbox
+                                label={"Perform a search for possible email addresses."}
+                                checked={checkedEmailaddress}
+                                onChange={(e) => setCheckedEmailaddress(e.currentTarget.checked)}
+                            />
+                            <Checkbox
+                                label={"Perform a TCP port scan"}
+                                checked={checkedPortscan}
+                                onChange={(e) => setCheckedPortscan(e.currentTarget.checked)}
+                            />
+                            {checkedPortscan && (
+                                <div style={{ marginLeft: "20px" }}>
+                                    <Checkbox
+                                        label={"Perform a TCP port scan showing output reporting filtered ports"}
+                                        checked={checkedFiltered}
+                                        onChange={(e) => setCheckedFiltered(e.currentTarget.checked)}
+                                    />
+                                    <Checkbox
+                                        label={"Read in the Banner received from the scanned port"}
+                                        checked={checkedBanner}
+                                        onChange={(e) => setCheckedBanner(e.currentTarget.checked)}
+                                    />
+                                    <TextInput
+                                        label={"Delay (in seconds)"}
+                                        placeholder={"Set the TTL in seconds when scanning a TCP port (Default 2)"}
+                                        type="number"
+                                        value={delay}
+                                        onChange={(e) => setDelay(parseInt(e.target.value))}
+                                    />
+                                </div>
+                            )}
+                        </>
+                    )}
+
+                    <Button type={"submit"}>Start Scanning</Button>
```

Comment:
Should have the tool's title. 
```suggestion
                    <Button type={"submit"}>Start { Title }</Button>
```